### PR TITLE
chain config for testnet

### LINKED ISSUE #67: Fork config for testnet
Instead of using `if ctx.GlobalBool(TestNetFlag.Name)` and `params.MainNetHomesteadBlock` everywhere move block configuration to testnet fork config.

### PR DESCRIPTION
**Pull request by splix**
fix #67

### CODE REVIEW

File: cmd/utils/flags.go [hunk 1]
```diff
@@ -840,20 +825,19 @@ func MustMakeChainConfigFromDb(ctx *cli.Context, db ethdb.Database) *core.ChainC
...
 			netsplitChoice = fmt.Sprintf("resulted in a network split (support: %t)", c.Forks[i].Support)
 		} else {
 			netsplitChoice = ""
 		}
-		testnetEnabled := sort.SearchStrings(testnet, c.Forks[i].Name) < len(testnet) && testnet[sort.SearchStrings(testnet, c.Forks[i].Name)] == c.Forks[i].Name
-		if !ctx.GlobalBool(TestNetFlag.Name) || testnetEnabled {
-			glog.V(logger.Warn).Info(fmt.Sprintf(" %v hard-fork at block %v %v", c.Forks[i].Name, c.Forks[i].Block, netsplitChoice))
+		pad := ""
+		if big.NewInt(1000000).Cmp(c.Forks[i].Block) > 0 {
+			pad = " "
```
Comment (elaineo) [inline cmd/utils/flags.go]: This is a little excessive, maybe cleaner to just have the logger leave an extra space

File: core/fork.go [hunk 2]
```diff
@@ -100,3 +100,22 @@ func LoadForks() []*Fork {
...
+			GasTable:     &params.GasTableHomestead,
+		},
+		&Fork{
+			Name:         "GasReprice",
+			Block:        big.NewInt(1783000),
+			NetworkSplit: false,
+			Support:      true,
+			GasTable:     &params.GasTableHomesteadGasRepriceFork,
+		},
+	}
```
Comment (elaineo) [inline core/fork.go]: Will we have testnets for other forks, or are you just listing the ones that are currently running?

File: cmd/utils/flags.go (hunk 1 above)
Comment (splix) [inline cmd/utils/flags.go]: Yeah, you caught me :) right, I should use formatting. 
fixed, see my next commit

File: core/fork.go (hunk 2 above)
Comment (splix) [inline core/fork.go]: it's just currently running. I'm going to add testnet for diehard also, so moving it all here

**Pull request merged**